docs/source/cpp/compute.rst

```diff
@@ -1929,3 +1929,28 @@ operation to the n-th and (n+abs(p))-th inputs.
   ``Subtract``. The period can be specified in :struct:`PairwiseOptions`.
 * \(2) Wraps around the result when overflow is detected.
 * \(3) Returns an ``Invalid`` :class:`Status` when overflow is detected.
+
+Swizzle functions
```

Comment:
Instead of creating a category for this, perhaps we should just document these functions as part of the "Selections" category? After all, "scatter" is quite similar to "take" except that it takes a reverse mapping of the indices.

Reply:
Agree.

Do you think if I can keep the term "swizzle" in the source code? It's used in source file names, comments, `RegisterVectorSwizzle` function name. I wanted to move them into the "selection" counterparts but that attempts just complicated the code organization pretty bad.

Reply:
Yes, it's ok to use "swizzle" in the source code IMHO.